app/scripts/components/projectComponents/addProjectMemberModal.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { PropTypes } from 'react';
+const { object, bool, array, string } = PropTypes;
+import { observer } from 'mobx-react';
+import mainStore from '../../stores/mainStore';
+import authStore from '../../stores/authStore';
+import AutoComplete from 'material-ui/AutoComplete';
+import CircularProgress from 'material-ui/CircularProgress';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+
+@observer
+class AddProjectMemberModal extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            errorText: null,
+            floatingErrorText: '',
+            value: null
+        }
+        this.search = _.debounce(this.search ,500);
+    }
+
+    componentDidUpdate() {
+        if(mainStore.toggleModal.open && mainStore.toggleModal.id === 'addMember') setTimeout(()=>this.fullName.focus(), 300);
```

Comment:
why not just set focus on `componentDidMount`? Without the need to check for a global mainStore mapping of which modal is shown